lib/scrub/autoscrub.rb

```diff
@@ -30,14 +39,17 @@ def initialize(path)
       @output_dir = @output_struct.date_subdir!("output")
       @log_dir = @output_struct.date_subdir!("log")
       @out_files = []
+      Services.register(:scrub_stats) { {} }
 
       # Once we have @member_id and @item_type,
-      # build a log path and re-register the service logger to log to that path
-      ymd = Time.new.strftime("%F")
+      # build a new log path and re-register the service logger to that path
+      ymd = Time.new.strftime("%F").delete("-")
       @logger_path = File.join(@log_dir, "#{@member_id}_#{@item_type}_#{ymd}.log")
       Services.logger.info "autoscrub logging to #{@logger_path}"
 
       Services.register(:scrub_logger) do
+        # Get the early log in there.
+        FileUtils.mv(early_scrub_logger_path, @logger_path)
```

Comment:
Well, I guess I answered my own question - the "early" stuff ends up getting moved to the "real" log once we know its name